src/Compilers/CSharp/Portable/Binder/SubsumptionDiagnosticBuilder.cs

```diff
@@ -129,6 +129,8 @@ private ErrorCode CheckSubsumed(BoundPattern pattern, DecisionTree decisionTree,
                         // the input can no longer be null. In that case a null pattern is subsumed.
                         if (isNull && !inputCouldBeNull)
                         {
+                            // Note: we do not have any test covering this. Is it reachable?
```

Comment:
>// Note: we do not have any test covering this. Is it reachable? [](start = 28, length = 64)

I am not sure how are we going to track the work of adding tests for this and other similar code paths if we are closing #9542 with this PR. Are we going to open different issue(s)?